.github/workflows/ci.yml

```diff
@@ -10,35 +10,43 @@ jobs:
   macos:
     runs-on: macOS-latest
 
+    strategy:
+      matrix:
+        xcode: ["12", "11.4"]
+
     steps:
       - name: Checkout
         uses: actions/checkout@v1
       - uses: actions/cache@v2
         with:
           path: .build
-          key: ${{ runner.os }}-spm-${{ hashFiles('**/Package.resolved') }}
+          key: ${{ runner.os }}-spm-xcode-${{ matrix.xcode }}-${{ hashFiles('**/Package.resolved') }}
           restore-keys: |
-            ${{ runner.os }}-spm-
+            ${{ runner.os }}-spm-xcode-${{ matrix.xcode }}-
       - name: Build and Test
         run: swift test -c release
         env:
-          DEVELOPER_DIR: /Applications/Xcode_11.4.app/Contents/Developer
+          DEVELOPER_DIR: /Applications/Xcode_${{ matrix.xcode }}.app/Contents/Developer
 
   linux:
     runs-on: ubuntu-latest
 
+    strategy:
+      matrix:
+        swift: ["5.3", "5.2"]
```

Comment:
I think this won't work, we'll need two different versions of SwiftSyntax used for each Swift version. The fact that SwiftSyntax can't be linked in statically (or redistributed with `swift-doc` itself) is a deal-breaker 😑

Reply:
We should be able to work around that, no? Could we use Swift version-specific manifests like `Package@swift-5.2.swift`?

Reply:
Spot on, I forgot about that trick, I'll add it to NSHipster/SwiftSyntaxHighlighter#8.